source/core/unix/slang-unix-cpp-compiler-util.cpp

```diff
@@ -0,0 +1,163 @@
+#include "slang-unix-cpp-compiler-util.h"
+
+#include "../slang-common.h"
+#include "../slang-process-util.h"
+#include "../slang-string-util.h"
+
+#include "../slang-shared-library.h"
+
+#include "../slang-io.h"
+
+// The method used to invoke VS was originally inspired by some ideas in
```

Comment:
This file isn't invoking VS, so the comment is a bit misleading. I assume the inspiration is still there, but this should refer to gcc/clang?